examples/provider-components/image_versions_patch.yaml

```diff
@@ -44,7 +20,7 @@ spec:
     spec:
       containers:
         - name: manager
-          image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:v0.3.16
+          image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:master
```

Comment:
Similar to the generate.sh comment: is there some tag to use here that does not change over time instead of master? I'd freezing to specific tag improves reproducibility and the chances that the examples won't stop working.

Reply:
Same goes here, I had to use images with master tag to be able to run examples generation successfully.

Reply:
@furkatgofurov7 have you tried with `k8s.gcr.io/cluster-api/kubeadm-bootstrap-controller:v0.4.0` image?

Reply:
Yes, that was tried, and it was not a successful deployment at the end compared to what I have using master.